src/shared/components/alert/Alert.module.scss

```diff
@@ -1,15 +1,14 @@
 .box {
   position: fixed;
-  z-index: 100;
+  z-index: 200;
   inset: 0;
-  background-color: rgb(0 0 0 / 30%);
```

Comment:
User can't click any elements except close button. So background opacity is preferred

Reply:
I fix it